index.coffee

```diff
@@ -6,8 +7,8 @@ class ClosureCompiler
   type: 'javascript'
   extension: 'js'
 
-  constructor: (config = {}) ->
-    @config = Object.assign @defaultFlags, config.plugins?.closurecompiler
+  constructor: (config) ->
```

Comment:
This is not necessary since few versions ago, `config` will be object with `plugins` object and other stuff. We will update docs accordingly soon enough.